test/parensBGone_test.js

```diff
@@ -0,0 +1,30 @@
+import { expect } from 'chai'
+import parensBGone from '../src/parensBGone'
+
+describe('parensBGone()', function() {
+  it('should be a function', function() {
+    expect(parensBGone).to.be.a('function')
+  })
+
+  context("given a string with extra parens", function() {
+
+
+    it("should return the string with only the extra parens removed", function() {
```

Comment:
If you're already defining the context for each one, the description for it could be a little bit more descriptive, because it's a little bit confusing to understand what's going on with all of the parentheses. I suppose that is just the nature of the problem though